scripts/release-train-issue-template.md

```diff
@@ -50,7 +50,7 @@ Wind down PR queue. There has to be enough time after the last (non-trivial) PR
 - [ ] Notify Telemetry / Play team to check against staged artifacts
 - [ ] Run a test against the staging repository to make sure the release went well, for example by using https://github.com/akka/akka-http-quickstart-scala.g8 and adding the sonatype staging repo with `resolvers += "Staging Repo" at "https://oss.sonatype.org/content/repositories/comtypesafe-xxx"`
 - [ ] Release the staging repository to Maven Central.
-- [ ] Checkout the newly created tag and run `sbt -Dakka.genjavadoc.enabled=true ++2.12.9 "deployRsync akkarepo@gustav.akka.io"` to deploy API and reference documentation.
+- [ ] Checkout the newly created tag and run `sbt -Dakka.genjavadoc.enabled=true ++2.12.9 publishRsync` to deploy API and reference documentation.
```

Comment:
Do we still want to fix that to 2.12.9 Scala version?

Reply:
I don't see why (but perhaps let's change that in a separate PR)